### DISCUSSION TITLE
Suggestion: Set up GithubPages

### DISCUSSION THREAD

**Discussion by sangheili868** [masupert/BeyondTheTorchlight#2] [Ideas]
Hi! I'm Tyrzaphir from the ttrpg penguin discord, and I use Github on a daily basis. I think its a great way to manage system rules, and one thing I think will really take it to the next level is setting up Github Pages to create a free URL that anyone can view your rules at. Having them already set up as markdown files makes it even easier. I think you can just follow this guide to get it going. Let me know if you need any help with this or using github!

https://nicolas-van.github.io/easy-markdown-to-github-pages/

**Comment by masupert** [comment]
Thanks. I have done some preliminary investigation on that. Would I reply pages into the same repository that I am storing my markdown files in or would I setup another repository for the html/css files?

**Comment by masupert** [comment]
Initial html landing page has been setup. Will continue to customize and publish files to it.
